src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ProcessorTestSuite.java

```diff
@@ -108,7 +105,8 @@ public void setUp(Construct<T> processorConstruct) throws Exception {
             cloudBlobClient,
             documentProcessor,
             envelopeProcessor,
-            errorWrapper,
+            envelopeRepository,
+            processEventRepository,
             SIGNATURE_ALGORITHM,
```

Comment:
We seem to always inject: envelopeProcessor, envelopeRepository and processEventRepository together. However we also inject the same 2 repositories in EnvelopeProcessor already. Couldn't that be simplified?